scripting/pvpoptin/utils.sp

```diff
@@ -77,7 +77,7 @@ void UpdatePvPParticles(int client) {
 	if (TF2_IsPlayerInCondition(client, TFCond_Disguised)) effectiveClass = view_as<TFClassType>(GetEntProp(client, Prop_Send, "m_nDisguiseClass"));
 	if (effectiveClass == TFClass_Unknown) effectiveClass = TF2_GetPlayerClass(client);
 	
-	if (tmask != clientParticleAttached[client] || clientForceUpdateParticle[client]) {
+	if (tmask != clientParticleAttached[client] && clientForceUpdateParticle[client]) {

```

Comment:
This looks wrong, the intent of the target mask `tmask` is to update players that on through the timer, in case e.g. someone joined the game. clientForceUpdateParticle is intended to bypass that on an event basis. Could you explain this a bit?